multimethod/__init__.py

```diff
@@ -15,7 +27,7 @@ def get_types(func: Callable) -> tuple:
     """Return evaluated type hints for positional required parameters in order."""
     if not hasattr(func, '__annotations__'):
         return ()
-    type_hints = typing.get_type_hints(func)
+    type_hints = get_type_hints(func)
```

Comment:
There was a mix of `import typing` and `from typing import ...` w/ some overlap. Figured I'd unify, but let me know if you'd prefer the other way (always use `import typing # as tp`).